eng/pipelines/release-go-images-pipeline.yml

```diff
@@ -146,6 +150,11 @@ extends:
                       -azdopat '$(System.AccessToken)'
                   displayName: ⌚ Wait for go-images build
                   timeoutInMinutes: 120
+
+                - script: |
+                    releasego generate-announcement -o $(Pipeline.Workspace)/AnnouncementData
```

Comment:
Hmm, to get this done even more incrementally, how about rebasing to remove my commits that change the yml for now? I think it would work fine to merge a PR with the tool first, then do the yml/pipeline integration in another PR.

Reply:
done